frontend/device/remarkable/device.lua

```diff
@@ -125,5 +126,161 @@ function Remarkable:reboot()
     os.execute("systemctl reboot")
 end
 
+local function osCapture(cmd, raw)
+    local f = io.popen(cmd, 'r')
+    if not f then
+        return nil
+    end
+    local s = f:read('*a')
+    f:close()
+    if not s then
+        return nil
+    end
+    if raw then
+        return s
+    end
+    s = string.gsub(s, '^%s+', '')
+    s = string.gsub(s, '%s+$', '')
+    s = string.gsub(s, '[\n\r]+', ' ')
+    return s
+end
+
+local function getNetworkProperty(path, name)
+    path = string.sub(path, string.len("/codes/eeems/oxide1/") + 1)
+    local json, err = rapidjson.decode(osCapture("rot --object Network:" .. path .. " wifi get " .. name))
+    return json
```

Comment:
You could simply pass it on. Or you could write the above as `local json = rapidjson.decode(osCapture("rot --object Network:" .. path .. " wifi get " .. name))`
```suggestion
    return json, err
```